Testscripts/Windows/PERF-STORAGE-MULTIDISK-RAID0-FIO.ps1

```diff
@@ -268,7 +268,19 @@ function Main {
             -OutPerfResultFile $outPerfResultFile
 
         $fioPerfResults = Get-FioPerformanceResults $outPerfResultFile
-
+        #Get current test modes from constants file
+        $modes = Run-LinuxCmd -ip $allVMData.PublicIP -port $allVMData.SSHPort `
```

Comment:
you can get the modes straight from the fioPerfResults, you don t need to do another ssh